ultra/ultra/rllib_train.py

```diff
@@ -149,6 +154,7 @@ def train(
 
     # Iteration value in trainer.py (self._iterations) is the technically the number of episodes
     for i in range(num_episodes):
+        print("i ASDASD", i)
```

Comment:
Is this supposed to be here?

Reply:
Whoops, now you see my debugging process.

Reply:
Haha, all good!